db/schema.rb

```diff
@@ -77,10 +77,10 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.string "status", null: false
-    t.bigint "school_contact_id"
-    t.string "will_need_chromebooks"
     t.string "school_or_rb_domain"
     t.string "recovery_email_address"
+    t.bigint "school_contact_id"
```

Comment:
@aldavidson you mind re-creating your DB so this noise goes away?

Reply:
i'll fix it

Reply:
fixed